packages/common/e2e/user-app-resources.e2e.ts

```diff
@@ -0,0 +1,302 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import {
+  ArcGISContextManager,
+  IUserHubSettings,
+  IUserSiteSettings,
+  UserResourceApp,
+  getUserHubSettings,
+  getUserSiteSettings,
+  setUserHubSettings,
+  setUserSiteSettings,
+} from "../src";
+import {
+  IAddUserResource,
+  getUserResource,
+  listUserResources,
+  removeUserResource,
+  setUserResource,
+} from "../src/utils/internal/userAppResources";
+import Artifactory from "./helpers/Artifactory";
+import config from "./helpers/config";
+import { getUser } from "@esri/arcgis-rest-portal";
+import { clearUserSiteSettings } from "../src/utils/internal/clearUserSiteSettings";
+import { clearUserHubSettings } from "../src/utils/internal/clearUserHubSettings";
+
+// NOTE: User App Resources is coupled with oAuth
+// so we can't simply create a session via username / pwd
+// because the resultant token is NOT tied to a clientId
+//
+// Auth into uar-harness-qa-pre-a-hub.hubqa.arcgis.com as paige_pa
+// ** BE SURE TO AUTH INTO THIS SITE** don't just transition from another site!
+// go into network tab and get the token
+const SITE_TOKEN =
+  "yK55bPcOmtP3tWOVxqO8CroEfutCJjZEcf_9CZiDEZjJL8CoSWbm6eaVjnIcrXg9-jgUam2s204haNHoi1JGd1YBLBtk6W5AQrEZVK77y8b-sX2LOVuc_pb0jhIJLYE_pLeqKa9AY46mVMU2pTWDrR_moQu73ih9P9lVi_1MV43l6tfK_9BLgDx7dcf8DsnM_TKTxsw8sdDFMPgIfGGOse-BKTMie31g_9Riyn1ir1I.";
+// site  "yK55bPcOmtP3tWOVxqO8CroEfutCJjZEcf_9CZiDEZjJL8CoSWbm6eaVjnIcrXg9-jgUam2s204haNHoi1JGd1YBLBtk6W5AQrEZVK77y8b-sX2LOVuc_pb0jhIJLYE_pLeqKa9AY46mVMU2pTWDrR_moQu73ih9P9lVi_1MV43l6tfK_9BLgDx7dcf8DsnM_TKTxsw8sdDFMPgIfGGOse-BKTMie31g_9Riyn1ir1I.";
```

Comment:
Did you meant to leave this duplicate token commented in?

Reply:
While debugging it's useful to have tokens bound to a site clientId, and one for arcgisonline clientId.  I'll remove them both